hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapMergePolicyTest.java

```diff
@@ -77,30 +69,38 @@ public void killAllHazelcastInstances() {
     @Parameterized.Parameter
     public ReplicatedMapMergePolicyTestCase testCase;
 
+    private TestHazelcastInstanceFactory factory;
```

Comment:
`HazelcastTestSupport` already has a `TestHazelcastInstanceFactory` which also stops the created HZ instances. Right now this seems to be missing in the test. I would remove the factory and use the provided methods from `HazelcastTestSupport`.